### PR TITLE
Employee home

### PR DESCRIPTION
**Pull request by AsalaKM**
**Employee home**

-  Add new rout for employee (/employee-home)
-  View all ticket that's related to the logged employee

**#6**

### CODE REVIEW

File: public/js/employee.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+/* global document */
+function move(x) {
```
Comment (FarahZaqout) [inline public/js/employee.js]: x is not a readable name.

File: src/controllers/employee.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+exports.get = (req, res, next) => {
+  const name = 'salwa';
+  getTicketByName(name)
+    .then((response) => {
+      const result = (response.rows);     
+      const resultSolved = [];
+      const resultWaiting = [];
+      result.forEach((x) => {
+        if (x.status_type === 'solved') {
```
Comment (FarahZaqout) [inline src/controllers/employee.js]: x is not a readable name

**Review by FarahZaqout — CHANGES_REQUESTED** [review]

File: public/js/employee.js (hunk 1 above)
Comment (OnsNader) [reply public/js/employee.js]: noted

**Review by OnsNader — APPROVED** [review]

**Review by FarahZaqout — CHANGES_REQUESTED** [review]
Login with employee credentials returns error 404

File: src/views/employee.hbs [hunk 3]
```diff
@@ -0,0 +1,99 @@
+ <section class="employee-section">
+    <div class="title">
+        <h2 class="greet">أهلا في مركز القطان للدعم الفني </h2>
```
Comment (FarahZaqout) [inline src/views/employee.hbs]: Qattan center is not a tech support center.

Should say something like: "tech support department of al qattan"

**Review by FarahZaqout — CHANGES_REQUESTED** [review]

**Review by FarahZaqout — APPROVED** [review]

**Pull request merged**